### PR TITLE
Enable cargo-spellcheck for blockchain/beacon 

### LINKED ISSUE #1685: Enable `cargo-spellcheck` for `blockchain/beacon`
**Issue summary**

Make sure that `cargo spellcheck` can run without errors in `blockchain/beacon`. For details, see the tracking issue.

**Other information and links**

### PR DESCRIPTION
**Pull request by tyshko5**
**Summary of changes**
Changes introduced in this pull request:
- 

**Reference issue to close (if applicable)**

Closes #1685

**Other information and links**

### CODE REVIEW

File: .config/forest.dic [hunk 1]
```diff
@@ -17,3 +17,7 @@ tuple
 mutex
 RPC
 parameterize/D
+Drand
+drand
+GRPC
+deserializing
```
Comment (lemmih) [inline .config/forest.dic:23]: You can use `de-serializing` instead.

File: .config/forest.dic [hunk 2]
```diff
@@ -17,3 +17,7 @@ tuple
 mutex
 RPC
 parameterize/D
+Drand
+drand
```
Comment (lemmih) [inline .config/forest.dic:21]: I'm leaning towards always referring to `drand` as a code block.

File: blockchain/beacon/src/drand.rs [hunk 3]
```diff
@@ -139,7 +139,7 @@ where
         prev: &BeaconEntry,
     ) -> Result<bool, anyhow::Error>;
 
-    /// Returns a BeaconEntry given a round. It fetches the BeaconEntry from a Drand node over GRPC
+    /// Returns a `BeaconEntry` given a round. It fetches the `BeaconEntry` from a Drand node over GRPC
```
Comment (lemmih) [inline blockchain/beacon/src/drand.rs:142]:
```suggestion
    /// Returns a `BeaconEntry` given a round. It fetches the `BeaconEntry` from a Drand node over [`gRPC`](https://grpc.io/)
```

File: blockchain/beacon/src/drand.rs [hunk 4]
```diff
@@ -33,7 +33,7 @@ impl DrandPublic {
     }
 }
 
-/// Type of the Drand network. In general only Mainnet and its chain information
+/// Type of the Drand network. In general only Main-net and its chain information
```
Comment (lemmih) [inline blockchain/beacon/src/drand.rs:36]:
```suggestion
/// Type of the `drand` network. In general only `mainnet` and its chain information
```

**Review by lemmih — CHANGES_REQUESTED** [review]

File: blockchain/beacon/src/drand.rs [hunk 5]
```diff
@@ -42,24 +42,24 @@ pub enum DrandNetwork {
...
+    /// URL endpoint to send JSON HTTP requests to.
     pub server: &'static str,
     /// Info about the beacon chain, used to verify correctness of endpoint.
     pub chain_info: ChainInfo<'a>,
     /// Network type
     pub network_type: DrandNetwork,
 }
 
-/// Contains the vector of BeaconPoints, which are mappings of epoch to the Randomness beacons used.
+/// Contains the vector of `BeaconPoints`, which are mappings of epoch to the `Randomness` beacons used.
```
Comment (lemmih) [inline blockchain/beacon/src/drand.rs:55]:
```suggestion
/// Contains the vector of `BeaconPoint`, which are mappings of epoch to the `Randomness` beacons used.
```

**Review by lemmih — APPROVED** [review]
Some consistency issues but those can be addressed later.

**Review by LesnyRumcajs — APPROVED** [review]

**Pull request merged**